src/integrationtest/java/com/ericsson/ei/frontend/CommonSteps.java

```diff
@@ -139,6 +140,16 @@ public void request_sent() throws Throwable {
         response = httpRequest.performRequest();
     }
 
+    @When("^request is sent til body \'(.*)\' is not received$")
```

Comment:
It is not the request that is sent? but response received? The time used here may also be added as an input. I would prepare wait {seconds} seconds for response body to contains {} data.
EI back-end maybe should return status code 204 if no data was found, 204 indicates that there was no content, so no aggregated object. Then this could wait for status code 200.

Reply:
I've made changes in the code that requires this pull request:
https://github.com/eiffel-community/eiffel-intelligence/pull/228